src/polyswarmd/config.py

```diff
@@ -310,8 +312,10 @@ def from_config_file(cls, filename):
         require_api_key = auth_uri is not None
         trace_transactions = config.get('trace_transactions', True)
         profiler_enabled = config.get('profiler_enabled', False)
+        redis_uri = config.get('redis_uri', os.environ.get('REDIS_URI', None))
```

Comment:
I'm not a fan of doing it via environment fallback. 
I think we should remain consistent and just do it via file/consul.

Reply:
This is consistent with AUTH_URI

Reply:
Yeah, I was copying from `AUTH_URI` there. And I agree, it has caught both @aarongooch and I before thinking setting an env var would make polyswarmd do something, but I think we should do a PR later that removes all these at once.